client/my-sites/plugins/plugin-management-v2/utils/get-plugin-action-statuses.ts

```diff
@@ -8,6 +8,7 @@ export const getPluginActionStatuses = ( state: AppState ) => {
 	const inProgressStatuses = getPluginStatusesByType( state, 'inProgress' );
 	const completedStatuses = getPluginStatusesByType( state, 'completed' );
 	const errorStatuses = getPluginStatusesByType( state, 'error' );
+	const updtoDateStatuses = getPluginStatusesByType( state, 'upToDate' );
```

Comment:
There is a typo here. It should be `upToDateStatuses` instead of `updtoDateStatuses`.